src/initialise_db.py

```diff
@@ -1,64 +1,73 @@
 '''
-Copyright (C) 2020 Link Shortener Authors
-Licensed under the MIT (Expat) License. See the LICENSE file found in the
-top-level directory of this distribution.
+Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
+Licensed under the MIT (Expat) License (see LICENSE in Documentation).
 '''
 from sanic import Blueprint
 
 from aiomysql import create_pool
 from aiomysql.sa import create_engine
 
-from sqlalchemy import MetaData, Table, Column, String, Integer, Boolean
+from sqlalchemy import MetaData, Table, Column, String
 from sqlalchemy.schema import CreateTable
 
 
 initdb_blueprint = Blueprint('intitialise_db')
 
 metadata = MetaData()
-initdb_blueprint.table = Table(
-    'links',
+initdb_blueprint.active_table = Table(
```

Comment:
I think we are going to need some ID and some timestamps in both tables but we can do it later...

Reply:
I couldn't make id work with SQLAlchemy. A primary key integer should be automatically set as autoincrement according to the documentation, but isn't. Specific autoincrement=True is ignored and omitted from the generated SQL query.

The table itself has to be created via CreateTable, otherwise the other SQLAlchemy commands don't always match or can't be linked to the database via pool/engine.

If I can find a way to autoincrement the id field, I'll add it in.